consul/src/main/scala/io/buoyant/consul/v1/BaseApi.scala

```diff
@@ -49,9 +49,15 @@ trait BaseApi extends Closable {
   private[v1] def mkreq(
     method: http.Method,
     path: String,
+    consistency: Option[ConsistencyMode],
     optParams: (String, Option[String])*
   ): http.Request = {
-    val params = optParams.collect { case (k, Some(v)) => (k, v) }
+    val consistencyMode = consistency match {
```

Comment:
`(consistencyMode.toSeq ++ optParams)` worked - it was failing to compile without `toSeq` :/